AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -236,34 +236,34 @@
 "accountdetails.title.newaccount" = "New Account";
 "accountdetails.title.editingaccount" = "Editing Account";
 "accounttype.cloud" = "Alfresco in the Cloud";
-"accounttype.alfrescoServer" = "Alfresco Server";
+"accounttype.alfrescoServer" = "Alfresco One";
```

Comment:
It is quite clear today that we have "One" and "Community Edition". We should use the neutral "Server" denomination. It's weak vs. "in the Cloud" but it works.